scripts/hero-artifacts.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+# Copyright (C) 2018 ETH Zurich and University of Bologna
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# HERO Github Artifacts
+#
+
+HERO_GITHUB_BASE_URL="https://github.com/pulp-platform/"
+
+#List of Artifacts URL
+HERO_TOOLCHAIN_GITHUB_ARTIFACT_NAME="hero-gcc-toolchain"
+
+#List of Version
+HERO_TOOLCHAIN_GITHUB_ARTIFACT_VERSION="v1.0.0"
```

Comment:
That's a nice solution. That part it is still really improvable and your solution it can be a good one.

In particular currently it miss also an automatic way to deploy from jenkins.